sources/packages/web/src/components/students/StudentApplicationsVersion.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-table>
+    <thead>
+      <tr>
+        <th id="submitted-header">Submitted</th>
+        <th id="status-header">Status</th>
+        <th id="actions-header">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-if="loading">
+        <td colspan="3">
+          <v-skeleton-loader type="table-row@2" />
+        </td>
+      </tr>
+      <tr v-for="version in versions" :key="version.id">
+        <td headers="submitted-header">
+          {{ getISODateHourMinuteString(version.submittedDate) }}
+        </td>
+        <td headers="status-header">
+          {{
+            mapApplicationEditStatusForStudents(version.applicationEditStatus)
```

Comment:
Are we deferring the use of status chip to upcoming PR or upcoming tickets related to the feature?

Reply:
There are no requirements or decisions on status chips or how they would look like so far.